metricbeat/module/aerospike/namespace/namespace_integration_test.go

```diff
@@ -26,20 +26,21 @@ import (
 
 	"github.com/elastic/beats/libbeat/tests/compose"
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
-	"github.com/elastic/beats/metricbeat/module/aerospike"
 )
 
 func TestData(t *testing.T) {
-	f := mbtest.NewReportingMetricSetV2Error(t, getConfig())
+	r := compose.EnsureUp(t, "aerospike")
+
+	f := mbtest.NewReportingMetricSetV2Error(t, getConfig(r.Host()))
 	if err := mbtest.WriteEventsReporterV2Error(f, t, ""); err != nil {
 		t.Fatal("write", err)
 	}
 }
 
 func TestFetch(t *testing.T) {
-	compose.EnsureUp(t, "aerospike")
+	r := compose.EnsureUp(t, "aerospike")
```

Comment:
nit: why `r`? would expect something more descriptive

Reply:
On the original PR `r compose.R` is a new object [passed to tests of suites](https://github.com/elastic/beats/pull/7957/files#diff-6fc9978e160cfcd0b98f66fa95884208R36), named after "runner control", to connect the tests with the runner. The name follows the "convention" of `testing.T`, `testing.B` and so on.
It is true that without the runner, and without being a parameter of tests it doesn't make much sense here, I'll replace this with something like `service`.

Reply:
Renamed to `service`.